tests/smoke.rs

```diff
@@ -6,7 +6,7 @@ use std::time::Instant;
 /// We allow for no change in state across this many consecutive iterations.
 const MAX_STATE_NOT_INCREMENTING: u8 = 1;
 /// We allow for no new transactions across this many consecutive iterations.
-const MAX_TXNS_NOT_INCREMENTING: u8 = 3;
+const MAX_TXNS_NOT_INCREMENTING: u8 = 5;
```

Comment:
Any idea why transactions are now incrementing at a slower rate?

Reply:
Not sure but I don't think that it is because of our changes. We can try this smoke test for main multiple times